teams/FriendMe.md

```diff
@@ -0,0 +1,6 @@
+#Developers
+* Leif Hasle, [Leif](https://github.com/Solarleaf)
```

Comment:
In the square brackets, put the username

Reply:
Should be updated